src/components/project/user/row.vue

```diff
@@ -20,7 +20,8 @@ except according to the terms contained in the LICENSE file.
       <form>
         <div class="form-group">
           <select class="form-control" :value="selectedRoleId"
-            :disabled="disabled" :title="selectTitle"
+            :aria-disabled="disabled || awaitingResponse"
+            :title="disabled ? $t('cannotAssignRole') : null"
```

Comment:
Light refactor here. I changed `disabled` so that it doesn't have anything to do with `awaitingResponse`, just whether the user is the current user. I think that simplifies things a little, including the `selectTitle` computed property. That computation is simple enough now that I think it can be in the template itself rather than a computed property.